package/com.unity.formats.usd/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [3.0.0-exp.5] - Unreleased
 ### Features
+- Added basic URP Import support.
```

Comment:
do I take your changes to mean that there is also basic URP export support now too?

Reply:
@SimonBoorer well, yes, but it's not dedicated URP export (like HDRP has), I just fixed the default shader export path to correctly export URP shaders too. So I wasn't sure if it counted as a feature 😅